Documentation/ThirdParties/Customers.md

```diff
@@ -0,0 +1,28 @@
+# Third Party Extension Support for Customers
+
+All existing third party extensions are compatible with the AEP Swift SDK. Using third-party extensions will need to use the backward compatibility layer and update how extensions are registered. This document will walk through the steps to get setup using third party extensions in the AEP Swift SDK.
+
+## Quick Setup
+
+#### Install Compatibility Layer
+
+Third party extensions requires the backward-compatible layer. Instructions on installing the backward compatible layer can be found [here](./Migration/ACP-Migration.md).
```

Comment:
`Existing Third party extensions require the backwards-compatibility layer.`

I think this is more accurate, right?